src/Microsoft.AspNetCore.Http.Connections/HttpConnectionDispatcher.Log.cs

```diff
@@ -40,6 +40,12 @@ private static class Log
             private static readonly Action<ILogger, Exception> _negotiationRequest =
                 LoggerMessage.Define(LogLevel.Debug, new EventId(10, "NegotiationRequest"), "Sending negotiation response.");
 
+            private static readonly Action<ILogger, HttpTransportType, Exception> _receivedDeleteRequestForUnsupportedTransport =
+                LoggerMessage.Define<HttpTransportType>(LogLevel.Trace, new EventId(11, "ReceivedDeleteRequestForUnsupportedTransport"), "Recieved DELETE request for unsupported transport: {TransportType}");
```

Comment:
i before e except after **whenever you feel like it**

Reply:
Ugh, that one is literally the bane of my existence.

Reply:
Still need to fix this, also, full stop